common/include/common_host_helpers.hpp

```diff
@@ -225,7 +225,7 @@ void print_device_matrix(std::ostream& os,
                          const rocblas_fill uplo = rocblas_fill_full,
                          const rocblas_int inca = 1)
 {
-    size_t to_read = std::max(inca * (m - 1) + m, lda * (n - 1) + n);
+    size_t to_read = std::max(inca * (m - 1) + n, lda * (n - 1) + m);
```

Comment:
Minor comment: perhaps multiplications such as "inca * (m-1)" or "lda * (n-1)" should be performed in 64bit arithemtic to avoid the potential for  32bit integer overflow? Perhaps these expressions can be coded using idx2D()?   

Just a thought.